src/uct/ib/base/ib_verbs.h

```diff
@@ -96,17 +96,6 @@ static inline struct ibv_mr *ibv_exp_reg_mr(struct ibv_exp_reg_mr_in *in)
 #  define IBV_EXP_ACCESS_ALLOCATE_MR                0
 #  define IBV_EXP_HAVE_CONTIG_PAGES(_attr)          0
 
-struct ibv_exp_reg_mr_in {
```

Comment:
Currently, the instrumentation is only available above verbs, so for testing I built with:
#define HAVE_MLX5_HW 0

I got conflicting definitions for these lines, and removing those seems to work for both cases.